packages/repco-server/src/lib.ts

```diff
@@ -5,11 +5,16 @@ import cors from 'cors'
 import express from 'express'
 import { PrismaClient } from 'repco-core'
 import Routes from './routes.js'
+import { createGraphqlHandler } from 'repco-graphql'
```

Comment:
import { createGraphqlHandler } from '../../repco-graphql/src/lib' //TO DO FIX IMPORT ERROR

Reply:
This fix is incorrect, we want to import from the packages directly. I think I found the reason why it wasn't working; the workspace declaration in the root package.json was incorrect. Trying the fix now.